tests/monitoring_test.go

```diff
@@ -458,6 +472,47 @@ var _ = Describe("[Destructive] Monitoring Tests", Serial, func() {
 			waitForNoPrometheusAlert(f, "CDIDataImportCronOutdated")
 		})
 
+		It("[test_id:XXXXX] DataImportCron failing metric expected value but no alert when no default storage class", func() {
```

Comment:
If we do go this route, maybe we could go for a unit test? this is at least a little expensive because of the cron and at most expensive because of Consistently

Reply:
Absolutely, great advise. Replaced the func test with much cheaper utest.

Reply:
Removing existing DIC functests and replacing them with utests can be done in a follow-up PR.

Reply:
Wouldn't remove all since we still want to know the interaction against the API server is going fine/alerts are fired